packages/did-session/src/sessionStore.ts

```diff
@@ -0,0 +1,53 @@
+import type { AccountId } from 'caip'
+import type { Cacao } from '@didtools/cacao'
+
+interface Session {
+  cacao: Cacao
+  keypair: CryptoKeyPair
+}
+
+export class SessionStore {
+  #db: IDBDatabase
+
+  constructor(db: IDBDatabase) {
+    this.#db = db
+  }
+
+  static async create(): Promise<SessionStore> {
+    if (typeof globalThis === 'undefined') {
```

Comment:
I think the check should be for accessing `indexedDB`, `globalThis` should be accessible in most environments, not only browsers.

```suggestion
    if (typeof globalThis.indexedDB === 'undefined') {
```